infer_segmentation.py

```diff
@@ -128,22 +157,35 @@ def main(gcp_bucket, stack_id, model_id, prediction_threshold):
         train_config['optimizer'],
         Path(local_model_dir, "model.hdf5").as_posix())
 
+    if image_ids is None:
+        images_list = list(Path(image_folder).iterdir())
+    else:
+        images_list = image_ids.split(',')
+
     n_images = len(list(Path(image_folder).iterdir()))
     for i, image_file in enumerate(sorted(Path(image_folder).iterdir())):
+        if image_file.parts[-1] in images_list:
+            print('Segmenting image {} of {}...'.format(i, n_images))
 
-        print('Segmenting image {} of {}...'.format(i, n_images))
+            image = Image.open(image_file)
 
-        image = Image.open(image_file)
+            segmented_image = segment_image(compiled_model, image, prediction_threshold,
+                                            target_size_1d, background_class, output_type)
 
-        segmented_image = segment_image(compiled_model, image, prediction_threshold, target_size_1d)
-
-        segmented_image.save(Path(output_dir, image_file.name).as_posix())
+            if 'labels' in output_type:
+                image_file_ext = image_file.name[-4:]
```

Comment:
this I'd suggest changing to `image_file.split('.')[-1]` -- assuming the extension is a fixed number of characters long will possibly hurt in the future (e.g., `jpeg`, `tiff`, `jpg`, `png`) but the string split should work more generally

Reply:
done thx!